ROS/osr_control/osr_control/roboclaw_wrapper.py

```diff
@@ -420,16 +421,94 @@ def stop_motors(self):
 
     def read_errors(self):
         """Checks error status of each motor controller, returns 0 if no errors reported"""
-        err = [0] * 3
+        err = ['0'] * 5
```

Comment:
error status has [length 3](https://github.com/nasa-jpl/osr-rover-code/blob/37b6033e2b718f5a65e54fffdc9d0a01897720c3/ROS/osr_interfaces/msg/Status.msg#L4)

```suggestion
        err = ['0'] * 3
```

Reply:
See commit ede2052